Ensure libintl.h can be found even if -I/opt/local/include is not in CPPFLAGS.
--- glib/gi18n-lib.h.orig	2017-07-13 18:03:39.000000000 -0500
+++ glib/gi18n-lib.h	2018-01-31 18:18:52.000000000 -0600
@@ -20,7 +20,7 @@
 
 #include <glib.h>
 
-#include <libintl.h>
+#include <@PREFIX@/include/libintl.h>
 #include <string.h>
 
 #ifndef GETTEXT_PACKAGE
--- glib/gi18n.h.orig	2017-07-13 18:03:39.000000000 -0500
+++ glib/gi18n.h	2018-01-31 18:18:56.000000000 -0600
@@ -20,7 +20,7 @@
 
 #include <glib.h>
 
-#include <libintl.h>
+#include <@PREFIX@/include/libintl.h>
 #include <string.h>
 
 #define  _(String) gettext (String)
